Gemfile

```diff
@@ -21,6 +21,7 @@ gem "omniauth-github", "1.1.2"
 gem "friendly_id", "~> 5.0.3"
 gem "turbolinks", "~> 2.3.0"
 gem "jquery-turbolinks", "~> 2.1.0"
+gem "sidekiq"
```

Comment:
nit: Also suggest putting the [`letter_opener`](https://github.com/ryanb/letter_opener) gem in the development group to view emails more easily.